imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -1,23 +1,23 @@
 """Methods for decomming packets, processing to level 1A, and writing CDFs for MAG."""
-
+from pathlib import Path
 
 from imap_processing.cdf.utils import write_cdf
 from imap_processing.mag.l0 import decom_mag
 
 
-def mag_l1a(packet_filepath, cdf_directory):
+def mag_l1a(packet_filepath, cdf_filepath):
     """
     Process MAG L0 data into L1A CDF files at cdf_filepath.
 
     Parameters
     ----------
     packet_filepath:
         Packet files for processing
-    cdf_directory:
+    cdf_filepath:
         Directory for output
```

Comment:
cdf_filepath is not super descriptive to me of what that is referring to (cdf inputs, outputs, IMAP base directory), if this is where you want the output to go I'd suggest `output_filepath` as a more explicit naming. But, not blocking at all on this.